### ISSUE TITLE
Dockerhub plan

### ISSUE THREAD

**Issue by joeribekker** [open-zaak/open-zaak#774]
Currently, Open Zaak and Open Notificaties uses Dockerhub to publish the images that are used by supplies/municipalities. Image pulls (downloads) are limited to 200 per 6 hours. We are soon to hit this limit. Also, stale images (old versions) will be deleted after a while.

We can do a few things:

1) Use a paid Dockerhub plan (budget needed)
2) "Force" CG/VNG to offer a solution for Open Source initiatives.
3) Other...

**Comment by LBozarov** [comment]
From Utrecht we gonna switch to Harbor

**Comment by sergei-maertens** [comment]
Switching to Harbor poses a similar problem - someone needs to host and maintain the Harbor installation, and the registry URLs must be extremely stable.

Harbor has the nice option of a pull-through cache configuration though.

**Comment by shazada** [comment]
Yes at our implementation at Gemeente A'dam, we sync/proxy the images into Harbor and don't have the hub.docker.com limit.

**Comment by sergei-maertens** [comment]
During grooming session we discussed options:

* host a Harbour instance (which needs to get funded) and retain old image tags there forever. Requires updates of documentation to point this out
* Update docs to mention that stale tags get marked stale/dissapear, but you can always build the image from github releases/tags. This would not require image repository hosting
* Get funding for docker hub
